gsa/src/gmp/commands/dashboards.js

```diff
@@ -80,18 +80,22 @@ class DashboardCommand extends GmpCommand {
       setting_id: id,
     }).then(response => {
       const {data} = response;
-      const {setting} = data.get_settings.get_settings_response;
+      let {setting} = data.get_settings.get_settings_response;
 
       if (!isDefined(setting)) {
         return response.setData({});
       }
 
+      if (isArray(setting)) {
+        setting = setting[0]; // use first setting to avoid crash if the same
+        // setting is returned twice in an array
```

Comment:
```suggestion
        // before https://github.com/greenbone/gvmd/pull/1106 it was possible that
        // a setting with the same UUID is added twice to the db
        // therefore use first setting to avoid crash
        setting = setting[0];
```